### DISCUSSION TITLE
Version 1.28.0

### DISCUSSION THREAD

**Discussion by apple502j** [ScratchAddons/ScratchAddons#4926] [Changelog]
## Bugfixes
- [#3805](https://github.com/ScratchAddons/ScratchAddons/issues/3805) blocks2image does not use editor-theme3 colors. (prio-3)
- [#3805](https://github.com/ScratchAddons/ScratchAddons/issues/3805) editor-theme3 does not affect block comment border colors. (prio-4)
- [#4434](https://github.com/ScratchAddons/ScratchAddons/issues/4434) better-featured-project button can overlap the my-ocular status. (prio-4)
- [#4494](https://github.com/ScratchAddons/ScratchAddons/issues/4494) Addon linking does not work for easter eggs. (prio-4)
- [#4494](https://github.com/ScratchAddons/ScratchAddons/issues/4494) Addon linking does not work when changing the hash in the URL after opening. (prio-4)
- [#4494](https://github.com/ScratchAddons/ScratchAddons/issues/4494) Addon linking can cause an error with invalid addon IDs. (prio-3)
- [#4494](https://github.com/ScratchAddons/ScratchAddons/issues/4494) Addon linking adds extra items to back-forward history. (prio-4)
- [#4807](https://github.com/ScratchAddons/ScratchAddons/issues/4807) editor-dark-mode shows wrong border color in RTL. (prio-4)
- [#4813](https://github.com/ScratchAddons/ScratchAddons/issues/4813) cloud-games uses a wrong project title as the preset. (prio-4)
- [#4817](https://github.com/ScratchAddons/ScratchAddons/issues/4817) editor-devtools find bar cannot search sounds. (prio-4)
- [#4817](https://github.com/ScratchAddons/ScratchAddons/issues/4817) editor-devtools find bar breaks with some locales. (prio-3)
- [#4829](https://github.com/ScratchAddons/ScratchAddons/issues/4829) editor-dark-mode loses background in the paint editor after switching locale. (prio-4)
- [#4829](https://github.com/ScratchAddons/ScratchAddons/issues/4829) onion-skinning settings disappear after switching locale. (prio-4)
- [#4830](https://github.com/ScratchAddons/ScratchAddons/issues/4830) better-emojis runs in all pages and causes performance issues. (prio-4)
- [#4869](https://github.com/ScratchAddons/ScratchAddons/issues/4869) Addon setting table rows with only one field is not centered. (prio-4)
- [#4885](https://github.com/ScratchAddons/ScratchAddons/issues/4885) Disabling a popup addon does not hide the currently open popups. (prio-4)
- [#4892](https://github.com/ScratchAddons/ScratchAddons/issues/4892) forum-time-zones contacts ScratchDB without proper privacy notice. (prio-3)
- [#4892](https://github.com/ScratchAddons/ScratchAddons/issues/4892) forum-time-zones does not work on topics with multiple pages. (prio-3)
- [#4903](https://github.com/ScratchAddons/ScratchAddons/issues/4903) hide-flyout hides the direction popup. (prio-3)
- [#4903](https://github.com/ScratchAddons/ScratchAddons/issues/4903) hide-flyout causes dragged sprites to be covered by sprite pane. (prio-3)
- [#4903](https://github.com/ScratchAddons/ScratchAddons/issues/4903) hide-flyout has a transparent border on the left while dragging. (prio-4)
- [#4903](https://github.com/ScratchAddons/ScratchAddons/issues/4903) editor-stage-left causes the right border to not work with hide-flyout and RTL. (prio-4)
- [#4903](https://github.com/ScratchAddons/ScratchAddons/issues/4903) columns puts the hide-flyout placeholder at incorrect location if using RTL. (prio-4)
- [#4909](https://github.com/ScratchAddons/ScratchAddons/issues/4909) columns rounds bottom-left corner too much if editor-stage-left is enabled. (prio-4)
- [#4910](https://github.com/ScratchAddons/ScratchAddons/issues/4910) search-profile shows the Profile button in red when dynamically disabled. (prio-4)
- [#4917](https://github.com/ScratchAddons/ScratchAddons/issues/4917) full-signature does not support scrolling for scratchblocks addon. (prio-4)
- [#4945](https://github.com/ScratchAddons/ScratchAddons/issues/4945) variable-manager does not validate variable names when renaming. (prio-3)
- [#4950](https://github.com/ScratchAddons/ScratchAddons/issues/4950) Some addon combinations reload already-loaded scripts. (prio-4)
- [#4965](https://github.com/ScratchAddons/ScratchAddons/issues/4965) forum-search incorrectly displays a forum name containing ampersand. (prio-4)
- [#4967](https://github.com/ScratchAddons/ScratchAddons/issues/4967) emoji-picker does not close the popup when the button is clicked again. (prio-4)
- [#4977](https://github.com/ScratchAddons/ScratchAddons/issues/4977) customize-avatar-border uses British English. (prio-4)
- [#4981](https://github.com/ScratchAddons/ScratchAddons/issues/4981) confirm-actions uses British English. (prio-4)
- [#5028](https://github.com/ScratchAddons/ScratchAddons/issues/5028) studio-followers allows username and icons to be selected. (prio-4)
- [#5036](https://github.com/ScratchAddons/ScratchAddons/issues/5036) block-cherry-picking can cherry-pick argument blocks from custom block definition block. (prio-3)

## New Features
- [#4449](https://github.com/ScratchAddons/ScratchAddons/issues/4449) cloud-games: add fullscreen button (prio-4)
- [#4553](https://github.com/ScratchAddons/ScratchAddons/issues/4553) discuss-button: add compact account drop down setting (prio-3)
- [#4759](https://github.com/ScratchAddons/ScratchAddons/issues/4759) clones: add show icon only setting (prio-3)
- [#4791](https://github.com/ScratchAddons/ScratchAddons/issues/4791) initialise-sprite-position: run on sprite duplication (prio-3)
- [#4803](https://github.com/ScratchAddons/ScratchAddons/issues/4803) forum-live-preview: add customizable refresh rate (prio-3)
- [#4817](https://github.com/ScratchAddons/ScratchAddons/issues/4817) find-bar and jump-to-def are now separated from editor-devtools. (prio-3)
- [#4819](https://github.com/ScratchAddons/ScratchAddons/issues/4819) New addon: remove-search-bar-autocomplete (prio-3)
- [#4881](https://github.com/ScratchAddons/ScratchAddons/issues/4881) New addon: zebra-striping (prio-3)
- [#4885](https://github.com/ScratchAddons/ScratchAddons/issues/4885) Make popups save the opened tab and open that by default next time. (prio-3)
- [#4891](https://github.com/ScratchAddons/ScratchAddons/issues/4891) confirm-actions: confirm project deletion (prio-3)
- [#4894](https://github.com/ScratchAddons/ScratchAddons/issues/4894) gamepad: add reset and clear buttons (prio-3)
- [#4916](https://github.com/ScratchAddons/ScratchAddons/issues/4916) editor-dark-mode: allow changing block palette opacity (prio-3)

## Changes
- [#3585](https://github.com/ScratchAddons/ScratchAddons/issues/3585) scratch-messaging: link to login page when logged off (prio-4)
- [#3585](https://github.com/ScratchAddons/ScratchAddons/issues/3585) scratch-messaging: always show reload button (prio-4)
- [#4478](https://github.com/ScratchAddons/ScratchAddons/issues/4478) Use declarativeNetRequestWithHostAccess when possible. (prio-4)
- [#4482](https://github.com/ScratchAddons/ScratchAddons/issues/4482) editor-devtools: add instruction to use Shift+Space on macOS (prio-4)
- [#4738](https://github.com/ScratchAddons/ScratchAddons/issues/4738) cloud-games: highlight currently open project (prio-3)
- [#4810](https://github.com/ScratchAddons/ScratchAddons/issues/4810) Change UI orange to blue in development versions. (prio-4)
- [#4813](https://github.com/ScratchAddons/ScratchAddons/issues/4813) cloud-games: add Massive Multiplayer Platformer preset (prio-4)
- [#4830](https://github.com/ScratchAddons/ScratchAddons/issues/4830) better-emojis: support dynamically enabling and disabling addon (prio-4)
- [#4840](https://github.com/ScratchAddons/ScratchAddons/issues/4840) exact-count: support dynamically enabling and disabling addon (prio-4)
- [#4892](https://github.com/ScratchAddons/ScratchAddons/issues/4892) forum-time-zones: support dynamically enabling and disabling addon (prio-4)
- [#4912](https://github.com/ScratchAddons/ScratchAddons/issues/4912) variable-manager: allow adding a list item with Enter key instead of Shift+Enter (prio-4)
- [#4949](https://github.com/ScratchAddons/ScratchAddons/issues/4949) Show error message if addon.json cannot be parsed. (prio-4)
- [#4964](https://github.com/ScratchAddons/ScratchAddons/issues/4964) default-costume-editor-color: remove beta tag (prio-3)

**Comment by WorldLanguages** [comment]
> forum-time-zones contacts ScratchDB without proper privacy notice

My bad, sorry! Looks like we forgot that single addon. Addons that use third party services need to state so in the addon description, and this wasn't the case.
